### DISCUSSION TITLE
Cannot disable Collaborative Editing

### DISCUSSION THREAD

**Discussion by SangeloDev** [outline/outline#3645] [Self Hosting]
Basically the title.
I've enabled it to try it out, then I couldn't disable it once I was done.
Any help would really be appreciated!

**Comment by Limezy** [comment]
Please have a look at the numerous discussions and issues about the exact same subject !

**Comment by himanshuparmar3355** [comment]
From Tag v0.62.1-multitest Features.tsx file content is 
```
         <Switch
          id="collaborativeEditing"
          name="collaborativeEditing"
          checked={data.collaborativeEditing}
          disabled={data.collaborativeEditing}
          onChange={handleChange}
        />
```
The switch button disabled will true when we activate this feature.
Is it done by mistake or for some other reason?

**Reply by tommoor** [reply]
As this has question has come up a lot the next version will allow disabling with a warning
